Editorial cleanup

igrigorik opened the pull request:
Outstanding TODO's:

- [x] fill in missing editor details for @cvazac.
- [x] sanity check IDL definitions and fix PerformanceResourceTiming reference.
- [ ] define hooks in NavigationTiming and ResourceTiming which should be called from processing (section 5.1). 
- ...


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/server-timing/cleanup.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/server-timing/c3d5d05...ecc25e8.html)

igrigorik commented:
@marcoscaceres can we bug you for a sanity check + some help on respec IDL front here? :)

igrigorik commented on index.html:
```diff
@@ -9,6 +9,13 @@
+      mailto: "cvazac@gmail.com",
+      company: "Akamai",
+      companyURL: "https://www.akamai.com/",
+      w3cid: "???"
```
@cvazac can you confirm + provide missing info?

cvazac replied:
`w3cid: 77116`
for `url`, you can list `https://twitter.com/vazac` or leave blank, please

marcoscaceres commented:
Happy to help.

cvazac commented:
Regarding hooks from resource and navigation timing, best I can think of is a note after the "Create" step of the "Processing Model" that says something like: "After the object has been created, and before it has been queued/added, allow other specs to add non-overlapping attributes." I couldn't think of a way (excepting pub/sub) to set up something that other specs could "call in to".

igrigorik commented:
@marcoscaceres thanks! In particular, any guidance on https://github.com/w3c/server-timing/pull/16/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR134 would be appreciated. If you have time to do an overall check, that would be awesome too. :-)

@cvazac thanks, updated the ID and removed the URL. Re, hook: we can do a similar approach as we have in HTML + Page Visibility specs:

> (HTML, [step 6](https://html.spec.whatwg.org/#unload-a-document)): Run any unloading document visibility change steps for Document that are defined by other applicable specifications.
> 
> (PV) If the user agent is to unload the Document, run the now hidden algorithm during the unloading document visibility change steps.

It may also make sense to define two hooks: one when entry is created, and one right before entry is queued, to allow for upstream specs to easily hook in and extend at both points.

siusin commented:
@igrigorik 
What is [L30 `format: "markdown"`](https://github.com/w3c/server-timing/blob/fe33ef954771c8033f477bbb9be30544b6dada89/index.html#L30) for here? Can it be removed? - it will create chaos for the HTML markup.

Let's publish a new WD for the WebPerf WG once this PR gets merged.

igrigorik commented:
@siusin same as https://github.com/w3c/preload/pull/104 -- this hasn't been a problem before?

marcoscaceres commented:
Sorry for the delay... starting review now.

igrigorik commented:
👍 thanks Marcos!

The last remaining bit here is to define a hook in RT and then update ST to use it. We should probably break that into a separate PR however.

@cvazac if the PR looks good, feel free to merge. I'll open a separate issue to track RT hook.

marcoscaceres commented:
I have a PR ready to go in my own fork to apply on top of this one to fix a few more xref nits.

cvazac commented:
Should/can we fix the travis failure before merging @igrigorik?

igrigorik commented:
@cvazac hm, not sure why that's not happy.. /cc @siusin. FWIW, I don't think we need to block on that. Merging, and @marcoscaceres can they apply additional fixes.
